protos/feast/core/CoreService.proto

```diff
@@ -112,6 +133,9 @@ message ListStoresResponse {
 }
 
 message ApplyFeatureSetRequest {
+    // Name of project that the feature set belongs to.
+    string project = 2;
```

Comment:
Will this becomes a mandatory parameter?

Reply:
Yes